easybuild/easyconfigs/p/PGI/PGI-2018a.eb

```diff
@@ -0,0 +1,16 @@
+easyblock = "Toolchain"
+
+name = 'PGI'
+version = '2018a'
```

Comment:
Hmm, what's the point here, why not use `PGI` `18.4` directly as a toolchain?

I don't really see the extra value in this...

Also, we kind of reserve the `<year><month>` versioning scheme for the common toolchains (& derivatives thereof). Maybe `2018.04` is better here?

Reply:
Ah, I didn't realise I could use `PGI/18.4` directly as a toolchain.  Do I just put `easyblock = "Toolchain"` at the top of `PGI-18.4-GCC-6.4.0-2.28.eb`?

Reply:
You don't need to, you can use the `PGI` module as is as a toolchain in easyconfigs:

```python
toolchain = {'name': 'PGI', 'version': '18.4-GCC-6.4.0-2.28'}
```

The `Toolchain` easyblock is just a fancy `Bundle` easyblock, it does little more than pack multiple modules together under one umbrella, but you don't need that here, since there's only `PGI`.

Reply:
Okay, thanks.  I didn't release this (or had forgotten!).  I'll remove `PGI-2018a.eb` file from this PR.